src/status_im/utils/ethereum/tokens.cljs

```diff
@@ -9,7 +8,10 @@
                :symbol   :ETH
                :decimals 18
                :icon     {:source (js/require "./resources/images/assets/ethereum.png")
-                          :style  (asset-border styles/color-light-blue-transparent)}})
+                          ;; TODO(goranjovic) find a better place to set UI info
+                          ;; like colors. Removed the reference to component.styles to
+                          ;; avoid circular dependency between namespaces.
+                          :style  (asset-border "#628fe333")}})
```

Comment:
why do we need this ns in styles? maybe better to delete it there?

Reply:
It is transitively needed via `platform` ns. It makes more sense to that styles should be aware of platform than that to have color settings in a util namespace.